pkgs/development/libraries/pipewire/default.nix

```diff
@@ -72,6 +73,26 @@ let
       ./pipewire-pulse-path.patch
       # Add flag to specify configuration directory (different from the installation directory).
       ./pipewire-config-dir.patch
+
+      # Various quality of life improvements that didn't make it into 0.3.22
```

Comment:
With this many patches should we just go to a revision on master? Releases are so frequent that we probably get 0.3.23 soon.

Reply:
That's what I was thinking. I still don't really understand how the module tests should work so chances are 0.3.23 is out before that anyway. Having the patches in here in the meanwhile helps those who want to test the PR in the meanwhile though, so no harm in keeping them around until upstream gets a new version.

Reply:
New version available: https://github.com/PipeWire/pipewire/commit/68f6c75caed047af32320ab4de0c06457457be54

Reply:
Note that upstream now starts pipewire-media-session via a systemd service, but Fedora opted to just uncomment the exec line in pipewire.conf instead: https://src.fedoraproject.org/rpms/pipewire/blob/rawhide/f/0001-conf-start-media-session-through-pipewire.patch

Reply:
The separate systemd service is much cleaner for the nixos module config and wireplumber seems to have one too, so I'll just use that.

Reply:
Ideal option would be for user to be able to choose which session manager he wants to use. Of course pipewire-media-session would be default. But for this to work we should wait for a new WirePlumber release, as 0.3.0 is more than half an year old.

Reply:
Yes that would be optimal. There are some tiny obstacles like no other session managers being packaged at all. But it is something I am trying to prepare for.